core/elf.ml

```diff
@@ -41,18 +41,10 @@ let create filename =
       let statically_mappable = is_non_pie_executable header in
       let debug = Owee_elf.find_section_body buf sections ~section_name:".debug_line" in
       Some { string; symbol; debug; base_offset; filename; statically_mappable }
-    | _, _ ->
-      Core.eprintf
-        "[Unable to find string or symbol table, will be unable to trigger on specific \
-         symbol. Was the binary built without debug info?]\n\
-         %!";
-      None
+    | _, _ -> None
   with
   | Owee_buf.Invalid_format _ ->
-    Core.eprintf
-      "[Invalid or unknown debug info format, will be unable to trigger on specific \
-       symbol.]\n\
-       %!";
+    Core.eprintf "Invalid or unknown debug info format.\n";
```

Comment:
Other places currently print errors/messages in `[ ... ]` (I assume to mirror `perf`). Do we want to drop that convention, or continue using it here?

Reply:
I kind of want to drop it, and also maaaaaaybe silence the perf output (I like seeing the trace size, but it's mostly noise).